app/src/main/java/org/mozilla/fenix/settings/wallpaper/WallpaperSettings.kt

```diff
@@ -67,6 +73,8 @@ fun WallpaperSettings(
     selectedWallpaper: Wallpaper,
     onSelectWallpaper: (Wallpaper) -> Unit,
     onViewWallpaper: () -> Unit,
+    tapLogoSwitchChecked: Boolean,
+    tapLogoSwitchOnCheckedChange: (Boolean) -> Unit
```

Comment:
nit: lets add docs for the new parameters :)

Reply:
Thanks for the reminder, adding some docs